source/includes/mongodb-compatibility-table-c.rst

```diff
@@ -15,6 +16,7 @@
      - MongoDB 2.6
 
    * - 1.19
+     -
```

Comment:
Is the C driver version 1.19 not compatible with MongoDB v5. while version 1.18 of the driver is compatible? No worries, if this is correct, but it seemed surprising at first glance.

Reply:
No, good catch.